### ISSUE TITLE
A11y_Azure Tools Storage Explorer_Accessible environment_Desired Storage Explorer theme_Name Role Value: Incorrect Role is assigned for Documentation and Troubleshooting Guide as buttons.

### ISSUE THREAD

**Issue by kupatkar99** [microsoft/AzureStorageExplorer#7624]
### Preflight Checklist

- [X] I have installed the [latest version of Storage Explorer](https://github.com/Microsoft/AzureStorageExplorer/releases/latest).
- [X] I have checked existing resources, including the [troubleshooting guide](https://docs.microsoft.com/azure/storage/common/storage-explorer-troubleshooting) and the [release notes](https://github.com/microsoft/AzureStorageExplorer/releases).
- [X] I have searched for [similar issues](https://github.com/Microsoft/AzureStorageExplorer/issues).

### Storage Explorer Version

1.32.1 (96)

### Architecture

x64

### Storage Explorer Build Number

20231114.10

### Platform

Windows

### OS Version

Windows 11 Enterprise Insider Preview (23H2)

### GitHub Tags:
#A11yMAS;#A11yTCS;#AILimited;#Win11;#DesktopApp;#A11ySev2;#BM_AzureToolsStorageExplorer_Win32_Dec2023;#Azure Tools Storage Explorer;#FTP#WCAG4.1.2;#NameRoleValue;

### Environment Details:
Application: Microsoft Azure Storage Explore
version: 1.32.1 (96)

### Steps to Reproduce

1. Launch Storage Explorer
2. Tab Navigate to "Get started" Tab and click on it.
3. Tab Navigate to "Documentation and Troubleshooting Guide".
4. Observe that Incorrect Role is assigned for Documentation and Troubleshooting Guide as buttons.

### Actual Experience

Incorrect Role is assigned for Documentation and Troubleshooting Guide as buttons.

### Expected Experience

Documentation and Troubleshooting Guide should be defined as Links.

### User Impact:
Screen reader users will face difficulty if Incorrect Role is assigned for Documentation and Troubleshooting Guide as buttons.

###Attachment:
![A11y_Azure Tools Storage Explorer_Accessible environment_Desired Storage Explorer theme_Name Role Value](https://github.com/microsoft/AzureStorageExplorer/assets/98530069/cbd10221-0032-4616-a1a6-4e4f2186dcd3)

**Comment by kupatkar99** [comment]
Verified the issue in below environment:
Microsoft Azure Storage Explorer
version: 1.33.0-dev(96)
Issue is not fixed. Please find below added attachment for reference.
![7624_Not fixed](https://github.com/microsoft/AzureStorageExplorer/assets/98530069/a0a10bc1-966b-4664-9aca-3ad94aff6657)

**Comment by kupatkar99** [comment]
Verified the issue in below environment:
Microsoft Azure Storage Explorer
version: 1.33.0-dev(96)
Issue is fixed. hence closing the bug. Please find below added attachment for reference.
![7624_Fixed](https://github.com/microsoft/AzureStorageExplorer/assets/98530069/fc94e5d2-f4a4-4b34-9437-3e5360c1481a)
